routes/task.routes.js

```diff
@@ -0,0 +1,8 @@
+import express from 'express';
+import {getTasks} from '../controllers/Task.controller.js';
+
+const router = express.Router();
+
+router.get('/all', getTasks);
+
```

Comment:
`All` can be omitted here, it is a default behavior. To get one you will use the id.